### DISCUSSION TITLE
v3: Neue LOG-Einträge Bedeutung?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#1020] [Q&A]
Welche Bedeutung haben die neuen Einträge `v3.0.0-alpha.80` im LOG:
![image](https://user-images.githubusercontent.com/84818934/133868686-94a81737-b3b6-4bb0-b56a-c591b551244b.png)

Vermutung, deswegen dauert es auch am Anfang länger?

**Comment by Zefau** [comment]
Waren für mich fürs Debugging. Die gehen wieder raus.
